CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## v0.44 [2022-08-12]
+
+_Bug fixes_
+
+- Fixed the `cis_v130_3_1` and `cis_v140_3_1` controls to correctly reference `cloudtrail_multi_region_read_write_enabled` query instead of `cloudtrail_enabled_all_regions` query. ([#479](https://github.com/turbot/steampipe-mod-aws-compliance/pull/479))
```

Comment:
```suggestion
- Fixed the `cis_v130_3_1` and `cis_v140_3_1` controls to correctly use the `cloudtrail_multi_region_read_write_enabled` query instead of the `cloudtrail_enabled_all_regions` query. ([#479](https://github.com/turbot/steampipe-mod-aws-compliance/pull/479))
```